### PR TITLE
Expanded on RichText.md

### PR DESCRIPTION
**Pull request by kasperosterbye**
I added some stuff on the rich test section under projects.

### CODE REVIEW

File: PharoProjects/RichText.md [hunk 1]
```diff
@@ -13,3 +13,39 @@ text
...
+## TextEmphasis
+The class TextEmphasis defines a set of simple (no font change) changes you can do. See the class side of TextEmphasis for the supported styles of emphasis.
+
+It i possible to overlap the different emphasis styles:
+
+```Smalltalk
+text := 'Hello World!' asText.
text
	addAttribute: TextEmphasis italic from: 7 to: 11;
	addAttribute: TextEmphasis struckOut from: 7 to: 11.
```
Comment (juliendelplanque) [inline PharoProjects/RichText.md]:
```suggestion
text := 'Hello World!' asText.
text
    addAttribute: TextEmphasis italic from: 7 to: 11;
    addAttribute: TextEmphasis struckOut from: 7 to: 11.
```

File: PharoProjects/RichText.md [hunk 2]
```diff
@@ -13,3 +13,39 @@ text
...
+
+You need to turn the font into a text attribute:
+
+```Smalltalk
+largeAttribute := TextFontReference
		toFont:
			(StrikeFont
				familyName: 'Bitmap Source Sans Pro'
				pointSize: 20).
'My larger text' asText addAttribute: largeAttribute from: 4 to: 10.
```
Comment (juliendelplanque) [inline PharoProjects/RichText.md]:
```suggestion
largeAttribute := TextFontReference
	toFont: (StrikeFont familyName: 'Bitmap Source Sans Pro' pointSize: 20).

'My larger text' asText addAttribute: largeAttribute from: 4 to: 10.
```

File: PharoProjects/RichText.md [hunk 3]
```diff
@@ -13,3 +13,39 @@ text
...
+## Clickable text.
+It is possible to make text do stuff when clicked. This is obviously a huge area. One simple example is:
+
+```Smalltalk
+textAction := TextAction new 
+	actOnClickBlock: [ 
+		Transcript nextPutAll: 'action clicked';cr;endEntry
+		].

'My action text' asText addAttribute: textAction from: 4 to: 10.
```
Comment (juliendelplanque) [inline PharoProjects/RichText.md]:
```suggestion
		].

'My action text' asText addAttribute: textAction from: 4 to: 10.
```

File: PharoProjects/RichText.md [hunk 4]
```diff
@@ -13,3 +13,39 @@ text
 ```
 
 If you inspect `text`, you will see that `Hello` is bold, `World` is red, `!` is green.
+
+## TextEmphasis
+The class TextEmphasis defines a set of simple (no font change) changes you can do. See the class side of TextEmphasis for the supported styles of emphasis.
```
Comment (juliendelplanque) [inline PharoProjects/RichText.md]:
```suggestion
The class `TextEmphasis` defines a set of simple (no font change) changes you can do. See the class side of `TextEmphasis` for the supported styles of emphasis.
```

File: PharoProjects/RichText.md [hunk 5]
```diff
@@ -13,3 +13,39 @@ text
...
+
+It i possible to overlap the different emphasis styles:
```
Comment (juliendelplanque) [inline PharoProjects/RichText.md]:
```suggestion
It is possible to overlap the different emphasis styles:
```

File: PharoProjects/RichText.md [hunk 6]
```diff
@@ -13,3 +13,39 @@ text
...
	addAttribute: TextEmphasis struckOut from: 7 to: 11.
+```
+
+## Colors
+Colored text is done as shown above. Examine the class `Color` for other colors.
+
+## Different fonts
+Loading and managing fonts is an issue in itself.
+
+You can see which fonts you have available in your image using 'StrikeFont actualFamilyNames'.
```
Comment (juliendelplanque) [inline PharoProjects/RichText.md]:
```suggestion
You can see which fonts you have available in your image using `StrikeFont actualFamilyNames`.
```

File: PharoProjects/RichText.md [hunk 7]
```diff
@@ -13,3 +13,39 @@ text
...
+
+```Smalltalk
+largeAttribute := TextFontReference
		toFont:
			(StrikeFont
				familyName: 'Bitmap Source Sans Pro'
				pointSize: 20).
'My larger text' asText addAttribute: largeAttribute from: 4 to: 10.
+```
+This should give you 'My larger text' with 'larger' being in font size 20 and in the 'Bitmap Source Sans Pro' font.
```
Comment (juliendelplanque) [inline PharoProjects/RichText.md]:
```suggestion

This should give you 'My larger text' with 'larger' being in font size 20 and in the 'Bitmap Source Sans Pro' font.
```

**Comment by kasperosterbye** [comment]
Cool Julien. I will need to find out where the windows things came from, I am on a mac.

**Pull request merged**